backend/Squirrel.ConsoleApp.BL/Services/ConnectionService.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Squirrel.ConsoleApp.BL.Exceptions;
+using Squirrel.ConsoleApp.BL.Interfaces;
+using Squirrel.ConsoleApp.Models;
+
+namespace Squirrel.ConsoleApp.BL.Services;
+
+public class ConnectionService: IConnectionService
+{
+    private readonly IConnectionStringService _connectionStringService;
+    private readonly IConnectionFileService _connectionFileService;
+    private readonly IClientIdFileService _clientIdFileService;
+
+    public ConnectionService(IConnectionStringService connectionStringService,
+        IConnectionFileService connectionFileService, IClientIdFileService clientIdFileService)
+    {
+        _connectionStringService = connectionStringService;
+        _connectionFileService = connectionFileService;
+        _clientIdFileService = clientIdFileService;
+    }
+
+    public string TryConnect(ConnectionStringDto connectionStringDto)
+    {
+        // we need IntegratedSecurity = true to connect
```

Comment:
it seems that now time to update it

```
// we need IntegratedSecurity = true to connect
// to MSSQL local DB (it will be changed as SettingController updates)
connectionStringDto.IntegratedSecurity = true;
```